.circleci/config.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     <<: *defaults
     steps:
       - attach_workspace: *attach_options
-      - run: npm run test -- --code-coverage --full
```

Comment:
Maybe we can move the code coverage to Bazel tests. But I don't know if it will support async/await there either.

What breaks btw? If we don't use the code coverage facility we might as well remove the code for it, since it will rot over time unused.

Also cc @hansl since he was that one that added code coverage in the first place.

Reply:
The code coverage reports are neither displayed or saved so running with it only served to increase the run time.

The version of Istanbul used in the repository is outdated and doesn't support newer ES features.  Upgrading code coverage would probably be a good followup.